critiquebrainz/frontend/templates/review/browse.html

```diff
@@ -5,24 +5,36 @@
 
 {% block content %}
 <h2>{{ _('Reviews') }}</h2>
-<p><em class="text-muted">{{ _('Ordered by creation date') }}</em></p>
+<p><em class="text-muted">{{ _('Ordered by') }} {{ sort_options[sort, sort_order] }}</em></p>
```

Comment:
We should have a check somewhere in the view to ensure that these two variables sort and sort_order are known and present in sort_options, otherwise if they're invalid this item will crash